src/GovUk.Education.ExploreEducationStatistics.Data.Model/Repository/BoundaryLevelRepository.cs

```diff
@@ -18,9 +19,32 @@ public BoundaryLevelRepository(StatisticsDbContext context)
             _context = context;
         }
 
+        public async Task<IEnumerable<BoundaryLevel>> Get()
+        {
+            return await _context.BoundaryLevel.ToListAsync();
+        }
+
         public Task<BoundaryLevel?> Get(long id)
         {
-            return _context.BoundaryLevel.SingleOrDefaultAsync(level => level.Id == id);
+            return id == 0
```

Comment:
Why is it important that the id not be 0?

Reply:
`0` implies a value wasn't provided, the default for a numeric value type.

Reply:
I agree with @mmyoungman, this seems a bit strange to me. I'd expect trying to get an id that's not found in the database to ultimately return a 404 rather than an exception.